entrypoint.sh

```diff
@@ -62,7 +62,7 @@ wait_for_redis() {
 if [ "$1" = 'afp-scheduler' ]; then
   wait_for_postgres
   wait_for_redis
-  gosu "${USER}" airflow initdb
+  # gosu "${USER}" airflow initdb
```

Comment:
Can I suggest to opt for checking of some env var (e.g. `AIRFLOW_SCHEDULER_NO_INITDB`), if not set to `"true"`, we run this command instead (i.e. opt-out)?

Rationale being this would help to reduce breakage, where possibly other users might have already expected it to always `initdb`.

Reply:
This was a lesson learnt from me when I upgraded to 1.10.2. This initdb command updates the db schema to the latest version and it is not quite possible to roll back if there's a need to. I'd rather force the initdb be done manually.

Reply:
Ah okay I had similar issue before too long ago, had to use `alembic downgrade` command. In that case, I agree with this decision.

But can I suggest to still allow `initdb` to be run as an opt-in option (e.g. `[[ "${AIRFLOW_SCHEDULER_INITDB}" == "true" ]]`?) Then by default, we simply don't set this env var to prevent accidental `initdb`.

Reply:
If we follow the above, I might also suggest to add env var `AIRFLOW_SCHEDULER_INITDB: "true"` into `docker-compose.override.yml`.

Rationale: previously `docker-compose up` simply works to allow the user to try and get a local demo version set-up running, and so to preserve this, we would have to add this env var, but only for `docker-compose.override.yml`. I would probably plastic the env var with comments about not turning it on to prevent airflow metadata version from auto-upgrading because it becomes very hard to downgrade.

Reply:
Okay sure

Reply:
A bit occupied with work these days so a bit late addressing this.

Please see if the change looks good to you.

Reply:
Okay tested `docker-compose up --build`, works just fine, thanks for this!